src/pykeen/models/mocks.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+
+"""Mock models that return random results.
+
+These are useful for baselines.
+"""
+
+import torch
+
+from .base import EntityRelationEmbeddingModel, Model
+from ..nn import EmbeddingSpecification
+from ..triples import CoreTriplesFactory
+
+__all__ = [
+    "MockModel",
+]
+
+
+class MockModel(EntityRelationEmbeddingModel):
```

Comment:
Rename to `RandomScoreModel`?

Reply:
Maybe if we do what you said above we might want to pick a different name

Reply:
Name changed in 5f8dfc6a